HelloMixpanel/HelloMixpanelTests/MixpanelOptOutTests.m

```diff
@@ -0,0 +1,213 @@
+//
+//  MixpanelOptOutTests.m
+//  HelloMixpanelTests
+//
+//  Created by Zihe Jia on 3/15/18.
+//  Copyright © 2018 Mixpanel. All rights reserved.
+//
+
+#import "MixpanelBaseTests.h"
+#import "MixpanelPrivate.h"
+#import "TestConstants.h"
+#import "MixpanelPeoplePrivate.h"
+
+@interface MixpanelOptOutTests : MixpanelBaseTests
```

Comment:
We could add some other useful test cases. For example:
- Init the SDK with optOutTracking=false and make sure that no events are sent or in the queue (this test will fail right now - see comments below)
- make sure that when the user opts in, we have an `$opt_in` event in the queue only